packages/cspell-lib/src/index.ts

```diff
@@ -27,16 +27,14 @@ export { Text };
 
 import * as ExclusionHelper from './exclusionHelper';
 export { ExclusionHelper };
-export { ExcludeFilesGlobMap, ExclusionFunction, Glob } from './exclusionHelper';
+export { ExcludeFilesGlobMap, ExclusionFunction } from './exclusionHelper';
 
 export { getLanguagesForExt } from './LanguageIds';
 export * from './trace';
 
 import { clearCachedFiles } from './Settings';
 import { refreshDictionaryCache } from './SpellingDictionary';
 
-export async function clearCachedSettings(): Promise<void> {
+export async function refreshCachedSettings(): Promise<void> {
```

Comment:
This is an external interface change. This will break the VS Code spell checker.

Reply:
This was a manual hack, because it shadows the export from Settings

Reply:
Good point.
This is the version that should be used. Maybe it is time to be a bit more explicit on what is exported from settings.